modules/kubernetes-addons/main.tf

```diff
@@ -268,7 +268,6 @@ module "yunikorn" {
   count             = var.enable_yunikorn ? 1 : 0
   source            = "./yunikorn"
   helm_config       = var.yunikorn_helm_config
-  irsa_policies     = var.yunikorn_irsa_policies
```

Comment:
why we removing this? `irsa_policies` not being used at all in the module?

Reply:
no sir - not in use